index.html

```diff
@@ -145,7 +143,7 @@ <h1>Professional Art Printing Data</h1>
         </div>
       </section>
     </div>
-    <footer>
+    <div>
```

Comment:
I would suggest using HTML Semantics in the HTML code. It would make your code easy to understand.

Reply:
- [ ] this is a good tip to give to a student 👍 however try to be specific to which part of the code you are referring to and what you want them to change exactly, it might not be clear to the student what you're expecting from them.

Reply:
Great you almost did it, I would suggest you, using HTML Semantics such as `<footer>` tag for the footer of your project. It would make your code easy to understand.